exec/plugin-commands-script-runners/test/runRecursive.ts

```diff
@@ -581,7 +581,8 @@ test('testing the bail config with "pnpm recursive run"', async () => {
       recursive: true,
       selectedProjectsGraph,
       workspaceDir: process.cwd(),
-    }, ['build', '--no-bail'])
+      bail: false,
+    }, ['build'])
```

Comment:
This test began failing after the conversion to use `@pnpm/test-ipc-server` since the `--no-bail` script was being passed to scripts.

Changing how `--no-bail` is configured to fix this test.